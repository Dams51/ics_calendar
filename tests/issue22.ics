BEGIN:VCALENDAR
PRODID:-//TOM//EN
VERSION:2.0
METHOD:PUBLISH
BEGIN:VEVENT
DTSTART:20200121T124500Z
DTEND:20200121T131500Z
DTSTAMP:20200213T202351Z
UID:5go0u0n5ghfa91rvbsktblt9ti@google.com
CREATED:20200116T165452Z
DESCRIPTION:Betws-y-Coed (Caravan Holiday)
LAST-MODIFIED:20200116T165553Z
LOCATION:Rm 12
SEQUENCE:0
STATUS:CONFIRMED
SUMMARY:Betws-y-Coed (Caravan Holiday)
TRANSP:OPAQUE
END:VEVENT
END:VCALENDAR
