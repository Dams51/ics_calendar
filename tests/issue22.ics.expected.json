[
    {
        "summary": "Betws-y-Coed (Caravan Holiday)",
        "description": "Betws-y-Coed (Caravan Holiday)"
    }
]
